lib/IO/Socket/Async/SSL.pm6

```diff
@@ -17,6 +18,12 @@ sub SSL_CTX_load_verify_locations(OpenSSL::Ctx::SSL_CTX, Str, Str) returns int32
     is native(&gen-lib) {*}
 sub SSL_get_verify_result(OpenSSL::SSL::SSL) returns int32 is native(&gen-lib) {*}
 
+sub d2i_PKCS12(Pointer, CArray[CArray[uint8]], long) returns Pointer is native(&gen-lib) {*}
+sub PKCS12_parse(Pointer, Str, CArray[Pointer], CArray[Pointer], CArray[Pointer])
+    returns int32 is native(&gen-lib) {*}
+sub OPENSSL_add_all_algorithms_noconf() is native(&crypto-lib) {*}
+sub OpenSSL_add_all_algorithms() is native(&crypto-lib) {*}
```

Comment:
These two add_all_algorithms functions appear to not be used.

Reply:
Okay, I'll take care of certificates then.
*_add_all_algorithms_* have to be used before PKCS12_parse() in try {} block, I remember doing that but cannot find in diff, probably I forgot to push something :/
I'll check that, thank you!